{
  "uid" : "64bd75a22b96ad54",
  "name" : "Find and delete user",
  "fullName" : "test.test_users.TestUsers#test_users",
  "historyId" : "476081d539132dfcd130f046186d34b7",
  "time" : {
    "start" : 1749296327354,
    "stop" : 1749296329363,
    "duration" : 2009
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}",
  "statusTrace" : "self = <test_users.TestUsers object at 0x7f82a76db5f0>\n\n    @pytest.mark.smoke\n    @allure.title(\"Find and delete user\")\n    def test_users(self):\n        # user = self.users_api.create_user()\n        # self.users_api.delete_user_by_id(user.uuid)\n>       self.users_api.get_list_all_users()\n\ntest/test_users.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/users/api.py:33: in get_list_all_users\n    model = self._validate_response(response, UsersListResponseModel)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.users.api.UsersAPI object at 0x7f82a76db2f0>\nresponse = <Response [401]>\nmodel = <class 'services.users.models.model_list_user.UsersListResponseModel'>\nstatus_code = 200, success = True\n\n    def _validate_response(self, response: requests.Response, model: type[BaseModel], status_code: int = 200,\n                          success: bool = True):\n        self.attach_response(response.json())\n        if success:\n>           assert response.status_code == status_code, response.json()\nE           AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}\n\nhelper/helper.py:20: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestUsers",
    "time" : {
      "start" : 1749296327353,
      "stop" : 1749296327353,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1749296327351,
      "stop" : 1749296327353,
      "duration" : 2
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}",
    "statusTrace" : "self = <test_users.TestUsers object at 0x7f82a76db5f0>\n\n    @pytest.mark.smoke\n    @allure.title(\"Find and delete user\")\n    def test_users(self):\n        # user = self.users_api.create_user()\n        # self.users_api.delete_user_by_id(user.uuid)\n>       self.users_api.get_list_all_users()\n\ntest/test_users.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/users/api.py:33: in get_list_all_users\n    model = self._validate_response(response, UsersListResponseModel)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.users.api.UsersAPI object at 0x7f82a76db2f0>\nresponse = <Response [401]>\nmodel = <class 'services.users.models.model_list_user.UsersListResponseModel'>\nstatus_code = 200, success = True\n\n    def _validate_response(self, response: requests.Response, model: type[BaseModel], status_code: int = 200,\n                          success: bool = True):\n        self.attach_response(response.json())\n        if success:\n>           assert response.status_code == status_code, response.json()\nE           AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}\n\nhelper/helper.py:20: AssertionError",
    "steps" : [ {
      "name" : "Get list users ",
      "time" : {
        "start" : 1749296327354,
        "stop" : 1749296329363,
        "duration" : 2009
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}\n",
      "statusTrace" : "  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/services/users/api.py\", line 33, in get_list_all_users\n    model = self._validate_response(response, UsersListResponseModel)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/workspace/helper/helper.py\", line 20, in _validate_response\n    assert response.status_code == status_code, response.json()\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "3434f2060ad38257",
        "name" : "API response",
        "source" : "3434f2060ad38257.json",
        "type" : "application/json",
        "size" : 127
      } ],
      "parameters" : [ {
        "name" : "offset",
        "value" : "0"
      }, {
        "name" : "limit",
        "value" : "10"
      } ],
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true,
    "stepsCount" : 1
  },
  "afterStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestUsers::0",
    "time" : {
      "start" : 1749296329399,
      "stop" : 1749296329399,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test User"
  }, {
    "name" : "tag",
    "value" : "smoke"
  }, {
    "name" : "parentSuite",
    "value" : "test"
  }, {
    "name" : "suite",
    "value" : "test_users"
  }, {
    "name" : "subSuite",
    "value" : "TestUsers"
  }, {
    "name" : "host",
    "value" : "7f7401e66739"
  }, {
    "name" : "thread",
    "value" : "7-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_users"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 2,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "9172ecdc1cbdf4ec",
        "status" : "failed",
        "statusDetails" : "AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}",
        "time" : {
          "start" : 1749295715104,
          "stop" : 1749295717609,
          "duration" : 2505
        }
      } ]
    },
    "tags" : [ "smoke" ]
  },
  "source" : "64bd75a22b96ad54.json",
  "parameterValues" : [ ]
}